src/index.js

```diff
@@ -14,26 +14,10 @@ const { Content } = Layout;
 const META_URL =
   'https://xlab-open-source.oss-cn-beijing.aliyuncs.com/open_leaderboard/meta.json';
 
-const getLastMonth = (lastUpdateTime) => {
-  let year = lastUpdateTime.getFullYear();
-  let monthIndex = lastUpdateTime.getMonth();
-  let date = lastUpdateTime.getDate();
-
-  if (date < 20) {
-    monthIndex--;
-  }
```

Comment:
Now the last month = the month before the lastUpdateTime's month, `date < 20` is no longer needed.